commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentMethodGroupRelsDisplayContext.java

```diff
@@ -206,9 +206,6 @@ else if (navigation.equals("inactive")) {
 					themeDisplay.getLocale()));
 		}
 
-		commercePaymentMethodGroupRelsCount +=
-			commercePaymentMethodGroupRels.size();
-
```

Comment:
got it, I don't really know why there is a sort few lines later, it was already there. I think either it could be removed without problem, consider that as of now we have only 4 payment method, or we can change the order by column in the service method

Reply:
Better I will use a service with OrderBy comparator parameter, it should override the default order by from service.xml